packages/slate-react/src/components/editable.tsx

```diff
@@ -1049,6 +1053,37 @@ export const Editable = (props: EditableProps) => {
   )
 }
 
+/**
+ * The props that get passed to renderPlaceholder
+ */
+export type RenderPlaceholderProps = {
+  contentEditable: boolean
+  children?: React.ReactNode
+  ref: React.RefObject<any>
+}
+
+/**
+ * Default placeholder style object.
+ */
+export const defaultPlaceholderStyle: React.CSSProperties = {
+  position: 'absolute',
+  pointerEvents: 'none',
+  width: '100%',
+  maxWidth: '100%',
+  display: 'block',
+  opacity: '0.333',
+  userSelect: 'none',
+  textDecoration: 'none',
+}
+
+/**
+ * The default placeholder render method
+ */
+
+export const defaultRenderPlaceholder = (props: RenderPlaceholderProps) => (
+  <span style={defaultPlaceholderStyle} {...props} />
+)
```

Comment:
I see what you mean about needing `forwardRef` to keep the resize logic. Is there any reason `forwardRef` wasn't working? It doesn't seem bad in my opinion to have a `<DefaultPlaceholder>` component that has ref-forwarding built in. And would stay consistent that way.

And then that `DefaultPlaceholder` component can be exported (like the others), and this little rendering function can stay private.

Reply:
Not sure what I did the last time but this of course absolutely works, no forwardRef required